questionGenerator.py

```diff
@@ -332,11 +336,42 @@ def getClusteredTopics(topicModeller, videoData=None):
         sys.exit("Topic clustering error. Exiting...")
 
     logging.info(f"Clustered Topics data shape: {clusteredTopics.shape}")
-    logging.info(f"Clustered Topics head: {clusteredTopics.head(5)}")
+    logging.info(f"Clustered Topics head:\n {clusteredTopics.head(3)}")
 
     return clusteredTopics
 
 
+def modifyDuplicateTopics(topicList):
```

Comment:
Why this step need to handled during the Q Generation? I feel this should be `TopicExtractor.py` so that `questionGenerator.py` just get the list. 

What do you think?

Reply:
This is because I consider the topic titles being generated within the `TopicModeller` class to be a final product of the topic extraction process, even if they are duplicates. This also is a byproduct of how BERTopic sort of holds data about the topic titles and associated models, where it is not as easy to change information within it to reflect changes for the topic titles without some significant reworking of the code. Even when the titles are the same, the actual topic data that made that cluster is different for both, with different keywords and relevant text being associated with each topic. 

So I just renamed it in the `questionGenerator.py` script as a way to acknowledge that this is a cosmetic change to the title over a change to the actual data held with the `TopicModeller`, in a way.

Reply:
ok